ui/packages/artalk/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.7.1",
   "license": "MIT",
   "description": "A Selfhosted Comment System",
+  "type": "module",
```

Comment:
No, this will introduce new issues in esm strict env, see my new opened discussion, a in time discussion would help.